Parse/PFObject.h

```diff
@@ -638,7 +638,7 @@ NS_REQUIRES_PROPERTY_DEFINITIONS
  @param objects The list of objects to fetch.
  @param error Pointer to an `NSError` that will be set if necessary.
  */
-+ (void)fetchAllIfNeeded:(PF_NULLABLE NSArray *)objects error:(NSError **)error;
++ (instancetype)fetchAllIfNeeded:(PF_NULLABLE NSArray *)objects error:(NSError **)error;
```

Comment:
This one should have an `NSArray` return type, since the block has an `NSArray` as a result.